client/java/src/main/java/io/openlineage/client/transports/ConsoleTransport.java

```diff
@@ -20,4 +20,16 @@ public void emit(OpenLineage.RunEvent runEvent) {
     // if DEBUG loglevel is enabled, this will double-log even due to OpenLineageClient also logging
     log.info(OpenLineageClientUtils.toJson(runEvent));
   }
+
+  @Override
+  public void emit(OpenLineage.DatasetEvent datasetEvent) {
```

Comment:
To make it work, we would need to introduce `Event` class/interface within a code that is generated from spec. To avoid hardcoding,  this should have been included within spec. So the json-schema would not only contain data definition, but also some hints on how to implement model classes.